content/docs/dvclive/api-reference/set_step.md

```diff
@@ -0,0 +1,54 @@
+# dvclive.set_step()
+
+Signals that the current step has ended and updates `step` with the given value.
```

Comment:
> updates `step` with the given value

Does it increase it by the given value? Sets it to the given value? Or defines the "step value" for which `step` will be increases with `next_step()`? Let's be more specific here to avoid speculation. Thanks

Reply:
This:
> Sets it to the given value directly

Updated to be more specific

Reply:
Thanks!